reducers/views/rhs.ts

```diff
@@ -350,6 +350,23 @@ function isMenuOpen(state = false, action: GenericAction) {
     }
 }
 
+function editChannelMembers(state = false, action: GenericAction) {
+    switch (action.type) {
+    case ActionTypes.SET_EDIT_CHANNEL_MEMBERS:
+        return action.active;
+    case ActionTypes.UPDATE_RHS_STATE:
+        if (!action.state) {
+            return false;
+        }
+        return state;
+
+    case UserTypes.LOGOUT_SUCCESS:
+        return false;
```

Comment:
Is this intended here?

Reply:
It seems that all reducers here took care of this case so I figured why not 😅